lib/compat/wordpress-6.5/interactivity-api/interactivity-api.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Interactivity API: Functions and hooks
+ *
+ * @package Gutenberg
```

Comment:
```suggestion
 * @package WordPress
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/58066/commits/bcefd095988267115f2fee15cb080ba0e1292dab.